lib/SearchAndSort/SearchAndSort.js

```diff
@@ -250,9 +250,13 @@ class SearchAndSort extends React.Component {
       stripes,
     } = this.props;
 
+    // Use local storage to remember whether the filter pane has been toggled off or not.
+    this.filterPaneVisibilityKey = '@folio/inventory/inventoryFilterPaneVisibility';
```

Comment:
I don't think the `@folio/inventory/...` key belongs at this level. Can this be configured dynamically in a similar way to how `appId` is constructed below?

Reply:
Yes, you're right. This should look better now.